packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+
+#### Headings
+
+<Notice variant="important" ariaLevel="3">
+
+It is **strongly** suggested that consumers add a heading element above their implementation of the Clinical Item View component. 
+
+The Clinical Item View structure consists of an unordered list filled with list items, it is vital we include a header to ensure that the list of displays have proper context within the webpage.
+For example - if an Item View contains contact information for a patient, you should add a header above the Item View that says something like 'Contact Information' or 'Patient Contact Information'.
+
+**A header not only acts as a label for the information being communicated to the end users, but also helps with visual and screenreader navigation throughout a page.**
+To best implement this header, consumers can use the Terra Core Heading component found [here](https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/heading/about)
+or the Clincial Header component found [here](https://engineering.cerner.com/terra-clinical/components/terra-clinical-header/clinical-header/clinical-header).
+
+Best practices for implementing headers are as follows:
+- Use one unique h1 per page that describes what that page is about. That h1 preferably starts just above the main content.
+- Use headings to describe the content below. Do not use an HTML heading just to make the text appear bigger or stand out.
+- Use heading levels like the index of a book: hierarchical.
+- Do not choose a heading by its size, but by its level in the context of the content.
+- Do not skip a heading level from the top down.
+
+More information on heading standards can be found [here](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/).
+
+Below you can find a couple examples of implementing a Terra Core Heading element above the Item View.
+
+</Notice>
+
+##### Code Examples
+
+<ItemViewWithHeader
+    title="ItemView - With Header"
+/>
+<ItemViewTwoColumnWithHeader
+    title="ItemView - Two Column Layout With Header"
+/>
+
+#### True Columns vs By Row
+
+The two column layout for the Clinical Item View has been updated to use true columns instead of rows formatted to look like columns. This affects not only the programmatic layout but also the screenreader's behavior in how these displays are read.
+The implementation of this is accomplished behind the `trueColumn` prop which defaults to true.
+
+The screenreader now reads through the first column, top to bottom, before moving on to the second column and reading its displays from top to bottom.
+This way of handling readout is better, as it is more natural for us to organize and read information in columns by reading them one at a time, top to bottom before reading through the next column.
+
+It is strongly recommended to utilize the true two column layout mentioned above, but if needed you can set the `trueColumn` prop to false which will revert to using the previous 'by row' logic.
```

Comment:
Shouldn't this be the other way to maintain passivity? (Yes I am still reviewing that PR 🥲 )

Reply:
Just made the changes to the other PR so here is the update: [20ea3c3](https://github.com/cerner/terra-clinical/pull/916/commits/20ea3c320ad857b187a59767160b2246a73d2bad)